.coafile

```diff
@@ -17,3 +17,7 @@ enabled = False
 files = **/*.rst, README.rst
 ignore = venv/**
 bears = InvalidLinkBear
+
+[rstcheck]
+bears = RSTcheckBear
+files = *.rst,**/*.rst
```

Comment:
Thanks . **.rst is equivalent of both. Checked on command prompt. By importing glob.